README.md

```diff
@@ -502,6 +489,30 @@ The `deploy` directory contains one directory or symlink for each
 tested Kubernetes release. The most recent one might also work on
 future, currently untested releases.
 
+- **Wait until all pods reach 'Running' status**
+
+```sh
+    $ kubectl get pods
+    NAME                    READY   STATUS    RESTARTS   AGE
+    pmem-csi-8kmxf          2/2     Running   0          3m15s
+    pmem-csi-bvx7m          2/2     Running   2          3m15s
```

Comment:
Why this driver restarts visible again :( ?

Reply:
not sure. When I made README addition, there was this text. I think this was Clear-based cluster. I haven't re-checked this properly. But as positive example, ubuntu-govm based cluster I develop right now under #230, shows clean state:
```
NAME                    READY   STATUS    RESTARTS   AGE
pmem-csi-controller-0   3/3     Running   0          8m46s
pmem-csi-node-5vx7t     2/2     Running   0          7m24s
pmem-csi-node-bqlbl     2/2     Running   0          7m51s
pmem-csi-node-ctjrq     2/2     Running   0          7m6s
```

I think this topic needs checking

Reply:
This README.md change isn't supposed to be in this PR at all.

Reply:
Let's get PR #234 merged and then I'll rebase.

Reply:
This spurious change was the result of incomplete rebasing. It is gone now.